traefik/values.yaml

```diff
@@ -349,6 +349,27 @@ persistence:
   annotations: {}
   # subPath: "" # only mount a subpath of the Volume into the pod
 
+certResolvers: {}
+#   letsencrypt:
+#     # for challenge options cf. https://doc.traefik.io/traefik/https/acme/
+#     email: email@example.com
+#     dnsChallenge:
+#       # also add the provider's required configuration under env
```

Comment:
```suggestion
We have to emphasize that Kubernetes Secret containing DNS provider data is mandatory.  Otherwise, a user might be confused about where the sensitive (APK_KEY, login) data should be added.